### ISSUE TITLE
CMake Error at ClangFormat-stamp with itk-5.1_beta

### ISSUE THREAD

**Issue by TheChymera** [InsightSoftwareConsortium/ITK#1304]
I am trying to bump our ITK package version to 5.1 beta, and it installs fine, but again I get an error ([build.log](https://github.com/InsightSoftwareConsortium/ITK/files/3703260/build.log)) when trying to test it.

This update is mainly so as to ensure compatibility with a newer ANTs version which I can actually build (apparently the build system needed a few new tweaks). Don't worry, my complaints will get less frequent after I manage to get some version of ANTs building :)

**Comment by dzenanz** [comment]
@TheChymera Your local test data cache is missing a file `NoPreambleDicomTest.dcm`, available from [here](https://data.kitware.com:443/api/v1/file/hashsum/SHA512/b262387856cc1b7f0f41e04736c237f6b8820cc444af6dce5507ca2ce00558dd9cdd22aa1d70bbe362416054e1225d9249524d12defa478f02c2f39a6be1fccc/download).

[ITKTestingData](https://github.com/InsightSoftwareConsortium/ITKTestingData) might need an update for Beta1. Is that correct @thewtex?

**Comment by thewtex** [comment]
> Your local test data cache is missing a file `NoPreambleDicomTest.dcm`, available from [here](https://data.kitware.com:443/api/v1/file/hashsum/SHA512/b262387856cc1b7f0f41e04736c237f6b8820cc444af6dce5507ca2ce00558dd9cdd22aa1d70bbe362416054e1225d9249524d12defa478f02c2f39a6be1fccc/download).
> 
> [ITKTestingData](https://github.com/InsightSoftwareConsortium/ITKTestingData) might need an update for Beta1. Is that correct @thewtex?

Yes -- [this patch](https://github.com/InsightSoftwareConsortium/ITK/pull/1342) will add the missing files for the next release.



> I get an error ([build.log](https://github.com/InsightSoftwareConsortium/ITK/files/3703260/build.log)) when trying to test it.

```
cd /var/tmp/portage/sci-libs/itk-5.1_beta/work/itk-5.1_beta_build/ClangFormat-prefix/src && /usr/bin/cmake -P /var/tmp/portage/sci-libs/itk-5.1_beta/work/itk-5.1_beta_build/ClangFormat-prefix/src/ClangFormat-stamp/ClangFormat-download-Gentoo.cmake && /usr/bin/cmake -E touch /var/tmp/portage/sci-libs/itk-5.1_beta/work/itk-5.1_beta_build/ClangFormat-prefix/src/ClangFormat-stamp/ClangFormat-download
1349 CMake Error at ClangFormat-stamp/ClangFormat-download-Gentoo.cmake:49 (message):                                                                                                                                                                                                                                                                                                                                                     
1350   Command failed: 1
1351 
1352    '/usr/bin/cmake' '-Dmake=' '-Dconfig=' '-P' '/var/tmp/portage/sci-libs/itk-5.1_beta/work/itk-5.1_beta_build/ClangFormat-prefix/src/ClangFormat-stamp/ClangFormat-download-Gentoo-impl.cmake'
1353 
1354   See also
1355 
1356     /var/tmp/portage/sci-libs/itk-5.1_beta/work/itk-5.1_beta_build/ClangFormat-prefix/src/ClangFormat-stamp/ClangFormat-download-*.log
```

@TheChymera This could be avoided by setting `-DITK_FORBID_DOWNLOADS:BOOL=ON`, but it is set to `OFF` in line 22. Is it possible to flip the value?

**Comment by thewtex** [comment]
All data issues should be resolved in ITK 5.1 RC 2. @TheChymera thanks for the report. If similar issues occur, re-opening or new issues are appreciated.
